openfisca_tools/model_api.py

```diff
@@ -136,8 +126,11 @@ def random(entity, reset=True):
     return x
 
 
-def is_in(values, *targets):
-    return sum(map(lambda target: values == target, targets))
+def is_in(values: ArrayLike, targets: list) -> ArrayLike:
+    return np.any([values == target for target in targets], axis=0)
+
+def between(values: ArrayLike, lower: float, upper: float) -> ArrayLike:
```

Comment:
Could you add an `inclusive` arg, like in [`pandas.between`](https://pandas.pydata.org/docs/reference/api/pandas.Series.between.html)?